config/playground/docker/dev/extra_config/osd.yml

```diff
@@ -0,0 +1,20 @@
+opensearch.hosts: [https://localhost:9200]
+opensearch.ssl.verificationMode: none
+opensearch.username: kibanaserver
+opensearch.password: kibanaserver
+opensearch.requestHeadersWhitelist: [authorization, securitytenant]
+opensearch_security.auth.anonymous_auth_enabled: true
+opensearch_security.multitenancy.enabled: false
+opensearch_security.multitenancy.tenants.preferred: [Global, Private]
+opensearch_security.readonly_mode.roles: [kibana_read_only]
+# Use this setting if you are running opensearch-dashboards without https
+opensearch_security.cookie.secure: false
+server.host: '0.0.0.0'
+# Use the consolidated menu and global header bar
+opensearchDashboards.branding.useExpandedHeader: false
+# Enable wizard feature
+wizard.enabled: true
+
+#csp settings
```

Comment:
what's the csp, can we provide full name here?

Reply:
Yeah, added full name for csp in [new commit](https://github.com/opensearch-project/dashboards-anywhere/pull/91/commits/afa1c7d57264e293228d81b98676ddfc36f9cac4)